test/models/terminal_executor_test.rb

```diff
@@ -65,13 +65,13 @@
 
     it "terminates the program on getpgid failures so we fail fast" do
       Process.expects(:getpgid).raises(Errno::ESRCH)
-      subject.execute!('echo "hi"').must_equal(nil) # status? of a killed process is 9
+      subject.execute!('sleep 1').must_equal(nil) # status? of a killed process is 9
     end
 
-    it "terminates the program on getpgid + kill failures when the program was already dead" do
+    it "handles the case when getpgid + kill both fail" do
```

Comment:
I like the original better `handles` is always weird :D 
`it "kills the program when xyz"`